src/backy/logging.py

```diff
@@ -157,8 +149,8 @@ def write(line):
             write("\n" + exc)
 
         # Log everything to the persistent log.
-        with open('/var/log/fc-qemu.log', 'a') as l:
```

Comment:
Huh. I'm surprised we're writing /var/log/fc-qemu.log here. This smells weird. Is this code even used?

Reply:
Ok, I'll merge this as apparently things work anyways ... o_O